.gitignore

```diff
@@ -111,6 +111,7 @@ ENV/
 env.bak/
 venv.bak/
 
+
```

Comment:
Seems you could rollback the change on .gitignore - an extra blank line?

Reply:
Can you please advice me how to do so?

Reply:
The simplest way in this case would just be to delete the blank line I think. That way when you commit it again I think the total number of files changed will not include this one since there is no net change.

Reply:
Thank you. fixed.